memory-viz/src/memory_model.ts

```diff
@@ -704,9 +738,6 @@ export class MemoryModel {
         height: number,
         style?: object
     ) {
-        if (style === undefined) {
```

Comment:
I'm not sure why you deleted this

Reply:
@yoonieaj this one is checking something different than the code you deleted above. If `style` is `undefined` then it needs to be set to _something_, perhaps an empty object, so that the next line (with the `...style`) executes successfully.